kapitan/cli.py

```diff
@@ -219,6 +219,13 @@ def main():
     init_parser.add_argument('--directory',
                              default=from_dot_kapitan('init', 'directory', '.'),
                              help='set path, in which to generate the project skeleton, assumes directory already exists. default is "./"')
+    init_parser.add_argument('--verbose', '-v', help='set verbose mode',
+                                action='store_true',
+                                default=from_dot_kapitan('init', 'verbose', False))
+    init_parser.add_argument('--target-name', help='provide a target-name ex: name1,name2',
```

Comment:
To keep behaviour consistent with other commands, turn this into `--targets/-t` and take an array instead of comma-separated.

Reply:
Ok. Understand, ignore my comment on target-name and compile-input.

Reply:
Ok. Understand, ignore my comment on target-name and compile-input.